client/redux/users.js

```diff
@@ -78,6 +78,7 @@ const deleteUser = (userId) => {
     try {
       const { data: user } = await axios.delete(`/api/users/${userId}`);
       dispatch(_deleteUser(user));
+      console.log(`Deleted ${userId}`);
```

Comment:
Don't forget to delete this console.log :)

Reply:
NOOO! I did the same thing, forgot to push after the commit